### DISCUSSION TITLE
Limiting number of tickets

### DISCUSSION THREAD

**Discussion by ghost** [alfio-event/alf.io#1271] [Q&A]
Hi! I'm Patricio, IT Manager of a students lead NGO from Argentina. I'm really thankful for the tool you created as it's really useful for us. I'm facing a problem with the settings of an event and I was wondering if someone could lend me a hand.
This event has multiple activities which require the attendant to book it in advance, as we have limited capacity. We had in mind to limit booking to one activity per attendant. For this, I created a different category per activity so each attendant can book their place when getting the ticket. Then, I tried limiting the number of tickets to 1 by setting "How many tickets can be purchased in a single reservation (default 5)" to 1 in event settings. However, I found out that this seems to limit the number of tickets per category. When testing, I found out that the same person could book one ticket for each category. Is there any solution for what I'm trying to accomplish?
Thanks in advanced

**Comment by cbellone** [comment]
this looks like a bug. I am creating an issue for this. 
Thank you for letting us know of this problem!

**Reply by ghost** [reply]
Thank you for your commitment! I would like to help with the project on a couple of weeks when I have more free time as it's very useful for us.
I could help with translation to begin as I'm a native Spanish speaker. Email templates could have translation and I think I can help on expanding the documentation both in English and Spanish, though my English should be reviewed my another person, just in case.

**Reply by cbellone** [reply]
> I could help with translation to begin as I'm a native Spanish speaker. 

That would be highly appreciated! Please follow the [how-to](https://github.com/alfio-event/alf.io/tree/master/src/main/resources/alfio/i18n#help-us-translate-alfio-into-your-language).
Spanish should already have a good coverage (see below) and has been translated by spanish organizers (meaning people living in Spain), but having an extra help wouldn't hurt at all :) 

> Email templates could have translation 

Email templates for the public (i.e. ticket buyer) are already internationalized. Email templates for organizers are in English and cannot be translated at the moment. This will change in the future when we'll have a multi-language Backoffice.

> and I think I can help on expanding the documentation both in English and Spanish, though my English should be reviewed my another person, just in case.

This would be great! The website theme we use (https://www.docsy.dev/docs/) supports multi-language content, so it should be doable.
Content will always be double-checked by us, don't worry!

Thank you!
